templates/vite/gitignore

```diff
@@ -0,0 +1,34 @@
+# Logs
```

Comment:
This looks like an inaccurate duplicate of the [actual `.gitignore`](https://github.com/FuelLabs/fuels-ts/pull/2942/files#diff-0254c1c034fe56942404fbdfad3296980635e29f1da3c9e6e153eef061649904)